src/Components/NavBar/NavBarMobileMenu/__tests__/NavBarMobileMenu.jest.tsx

```diff
@@ -169,7 +169,7 @@ describe("NavBarMobileMenu", () => {
 
     it("tracks link clicks", () => {
       const linkContainer = getMobileMenuLinkContainer("notAdmin")
-      linkContainer.find("a").first().simulate("click")
+      linkContainer.find("a")[2].simulate("click")
```

Comment:
`linkContainer.find("a").at(1).simulate("click")`

Reply:
`linkContainer.find("a").at(0)` - Inbox link (since `force-enable-new-activity-panel` is disabled)
`linkContainer.find("a").at(1)` - Buy link